test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -643,6 +659,41 @@ def _assert_artifact_free(output, stray_artifacts):
             artifact, output.stdout
         ), f"Matched {artifact} in {output.stdout} while running {output.command}"
 
+def _test_sm_toolkit_and_ts_version(image, region):
+    """
+    @param image: ECR image URI
+    Make sure SM inference toolkit and torchserve versions match docker image label.
+    """
+
+    cmd_smkit = "pip show sagemaker-pytorch-inference | grep -i Version"
+    cmd_ts = "torchserve --version"
+    ctx = Context()
+    container_name = get_container_name(
+        "pytorch-smtoolkit-ts-check", image)
+    start_container(container_name, image, ctx)
+    ts_version_from_output = None
+    toolkit_version_from_output = None
+    output_smkit = run_cmd_on_container(
+        container_name, ctx, cmd_smkit, executable="bash"
+    )
+    tk_match = re.search(r'(\d+\.\d+\.\d+)',str(output_smkit.stdout))
+    if tk_match:
+        toolkit_version_from_output = tk_match.group(0)
+    output_ts = run_cmd_on_container(
+        container_name, ctx, cmd_ts, executable="bash"
+    )
+    ts_match = re.search(r'(\d+\.\d+\.\d+)', str(output_ts.stdout)) 
+    if ts_match:
+        ts_version_from_output = ts_match.group(0)
+    image_labels = get_labels_from_ecr_image(image, region)
+    if toolkit_version_from_output != None and ts_version_from_output != None:
+        expected_label = f"com.amazonaws.ml.engines.sagemaker.dlc.inference-toolkit.{toolkit_version_from_output}.torchserve.{ts_version_from_output}"
+    else:
+       raise RuntimeError("The image label can't be determined because one of the toolkit or torchserve versions are null") 
```

Comment:
This error message will be fairly unhelpful, because it will require user to first determine which package is missing. Why not just raise an error right after you look for a match, signal that you can't find the package of interest, and also dump the output/stdout so that the user has more debug information?